opennms-webapp/src/main/webapp/WEB-INF/applicationContext-spring-security.xml

```diff
@@ -50,6 +50,11 @@
     <intercept-url pattern="/rest/classifications/**" method="DELETE" access="ROLE_REST,ROLE_ADMIN,ROLE_FLOW_MANAGER" />
     <intercept-url pattern="/rest/classifications/**" method="PUT" access="ROLE_REST,ROLE_ADMIN,ROLE_FLOW_MANAGER" />
 
+    <!-- Allow certain actions for the ROLE_DEVICE_CONFIG_BACKUP role -->
+    <intercept-url pattern="/rest/device-config/**" method="POST" access="ROLE_REST,ROLE_ADMIN,ROLE_DEVICE_CONFIG_BACKUP" />
+    <intercept-url pattern="/rest/device-config/**" method="DELETE" access="ROLE_REST,ROLE_ADMIN,ROLE_DEVICE_CONFIG_BACKUP" />
+    <intercept-url pattern="/rest/device-config/**" method="GET" access="ROLE_REST,ROLE_ADMIN,ROLE_DEVICE_CONFIG_BACKUP" />
```

Comment:
Can we add `PUT`, too. Just to play it safe.

Reply:
sure